client/my-sites/email/titan-new-mailbox-list/index.jsx

```diff
@@ -52,10 +53,18 @@ const TitanNewMailboxList = ( {
 	};
 
 	const onMailboxAdd = () => {
+		recordTracksEvent( 'calypso_email_titan_add_mailboxes_add_another_mailbox_button_click', {
+			mailbox_count: mailboxes.length + 1,
+		} );
+
 		onMailboxesChange( [ ...mailboxes, buildNewTitanMailbox( selectedDomainName, false ) ] );
 	};
 
 	const onMailboxRemove = ( currentMailboxes, uuid ) => () => {
+		recordTracksEvent( 'calypso_email_titan_add_mailboxes_remove_mailbox_button_click', {
+			mailbox_count: mailboxes.length - 1,
+		} );
+
```

Comment:
The mailbox count at that level may not necessarily be the same as 1 (one) less the current value. There is logic that may evaluate to something else , so it may be more accurate to use the `updatedMailboxes` value?

Reply:
Fair point! The risk of `updatedMailboxes` having a different length than `mailboxes.length - 1` seems quite slim, but it's a very minor change and increases correctness, so I pushed it 👍